src/components/extensive/Modal/ModalWithClose.tsx

```diff
@@ -35,6 +36,15 @@ const ModalCloseLogo: FC<ModalCloseProps> = ({
       </div>
 
       {children}
+      <When condition={!!iconProps}>
+        <Icon
+          {...iconProps!}
+          width={iconProps?.width || 40}
+          className={twMerge("mb-8", iconProps?.className)}
+          style={{ minHeight: iconProps?.height || iconProps?.width || 40 }}
```

Comment:
There are several examples here and in files above that are using `||` instead of `??`; please correct.

Reply:
done